automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -724,3 +714,19 @@ trait LeonardoTestUtils
     x => x.status == status || x.status == AppStatus.Error
 
 }
+
+// Ron and Hermione are on the dev Leo's allowed list, and Hermione is a Project Owner
+sealed trait TestUser extends EnumEntry with Product with Serializable {
```

Comment:
What are the properties of sealed trait that make this the approach we are using?

Reply:
The Scala compiler can check for exhaustiveness when a `trait` is `sealed` because `sealed trait`s can be `extend`ed only in the same file (see [here](https://underscore.io/blog/posts/2015/06/02/everything-about-sealed.html) for details). The exhaustiveness checking is probably not very important for this case but I think locality is desirable.

Reply:
I like this ADT.. but does it need to be an enum? If not using any enumeratum functionality like `withName` then maybe this should just be a regular trait.

Reply:
Doesn't need to be an enum. Thanks for the catch!